bhr.js

```diff
@@ -260,11 +291,19 @@ async function fetchHangs(requestedDate, bugsPromise) {
       continue;
     }
 
+    let annotationsKey = getAnnotationsKey(thread, id);
+    let annotations = annotationsMap.get(annotationsKey);
+    if (!annotations) {
+      annotations = getHangAnnotations(thread, id);
```

Comment:
Well, I'm curious about your take on this, because I have a suspicion that we will want to allow filtering hangs based on their annotations, at which point we do need to compute it up-front, and we may as well only take one code path to reduce complexity. Thoughts?

Reply:
If we are convinced we'll want to do this soon, then yes, we'll need to compute the annotation information up-front.

Reply:
It definitely seems useful to me to be able to filter to, say, only `Unrecovered` hangs. It's also pretty easy to implement, though we'd need to settle on a UI.